### PR TITLE
added signup login styling

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
Liked your UI

Suggestion:- On your home page there is a horizontal scrollbar coming so fix it

**Review by romabulani — APPROVED** [review]
- Code looks good to merge

Suggestion : on login Page, the button text is Signup, change it to Login

File: src/pages/Auth/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from 'react-router-dom';
+import '../auth.css';
+export const Login = () => {
+  const navigate = useNavigate();
```
Comment (PeeyushPrashant) [inline src/pages/Auth/Login/Login.jsx:4]: like the way you have used to navigate, instead of using "Link" tag

**Review by PeeyushPrashant — APPROVED** [review]
code looks modular
folder structure is easily understandable

**Review by vinc3nati — APPROVED** [review]

File: src/pages/Auth/Login/Login.jsx (hunk 1 above)
Comment (Rohanmond) [reply src/pages/Auth/Login/Login.jsx:4]: thanks

**Pull request merged**